docs/piloting.rst

```diff
@@ -107,7 +107,7 @@ An FTP client can also be used to view and copy flight plans on-board Bebop. `Fi
 
 Then open `Site Manager` (top left), click `New Site`:
 
-    * `Host`: 192.168.42.1
+    * `Host`: 192.168.42.11
```

Comment:
Are you sure about this change? Unless changed recently, `bebop` assigns `192.168.42.1` to itself by default.